reconciletags/cloudbuild.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
```

Comment:
This is a bit of an unusual YAML, so it might be worth explaining what it does via comments. You might want to rename it to e2e_cloudbuild.yaml as well, to make it clear this is just used for e2e tests.